src/utils/model_library.py

```
@@ -0,0 +1,632 @@
+        with self._lock:
+            record = self._load_manifest()["models"].get(model_id)
+            if record is None:
+                raise ModelNotFoundError(f"Unknown model ID: {model_id!r}")
+            return self._public_metadata(record)
+
+    def update_model(
+        self,
+        model_id: str,
+        *,
+        display_name: str | _Unset = _UNSET,
+        class_names: list[str] | tuple[str, ...] | None | _Unset = _UNSET,
```

Ruff (0.16.1)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)